_posts/2020-08-31-faf-rename.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "FAF is dead, long live ABRT Analytics"
+date:   2019-03-28 18:48:44 +0100
+author: Miroslav Suchý
+visible: 1
+categories: FAF Analytics
+---
+ABRT team decided to rename [FAF](https://github.com/abrt/faf/) to new name: **ABRT Analytics**.
+
+Disclaimer: this blogpost was written in 2019-03-28. But I forgot to publish it. Lets do that now. :)\
+
+There were several reasons. FAF stands for Fedora Analysis Framework, but it is for a long time not bound only to Fedora. While we support deployment to Fedora and RHEL/CentOS only, it can analyze crashes from any system or container.
```

Comment:
s/but it is for a long time not bound only to Fedora/but it's been a long time since it hasn't been bound to Fedora exclusively/  
  
(Somebody please suggest a better wording, I still don't really like this)

Reply:
What about:
FAF stands for Fedora Analysis Framework, but only very old versions of FAF were bound to Fedora.

The entire line would be:
There were several reasons. FAF stands for Fedora Analysis Framework, but only very old versions of FAF were bound to Fedora. While we support deployment to Fedora and RHEL/CentOS only, it can analyze crashes from any system or container.

Reply:
:+1: